Common/DataModel/OccupancyTables.h

```diff
@@ -0,0 +1,240 @@
+DECLARE_SOA_COLUMN(MeanOcc_NTrackSize_Unfm_80, meanOcc_NTrackSize_Unfm_80, float);
+DECLARE_SOA_COLUMN(MeanOcc_NTrackTPC_A_Unfm_80, meanOcc_NTrackTPC_A_Unfm_80, float);
+DECLARE_SOA_COLUMN(MeanOcc_NTrackTPC_C_Unfm_80, meanOcc_NTrackTPC_C_Unfm_80, float);
+DECLARE_SOA_COLUMN(MeanOcc_NTrackITS_TPC_Unfm_80, meanOcc_NTrackITS_TPC_Unfm_80, float);
+DECLARE_SOA_COLUMN(MeanOcc_NTrackITS_TPC_A_Unfm_80, meanOcc_NTrackITS_TPC_A_Unfm_80, float);
+DECLARE_SOA_COLUMN(MeanOcc_NTrackITS_TPC_C_Unfm_80, meanOcc_NTrackITS_TPC_C_Unfm_80, float);
+
+DECLARE_SOA_COLUMN(MeanOccRobust_T0V0Prim_Unfm_80, meanOccRobust_T0V0Prim_Unfm_80, float);
+DECLARE_SOA_COLUMN(MeanOccRobust_FDDT0V0Prim_Unfm_80, meanOccRobust_FDDT0V0Prim_Unfm_80, float);
+DECLARE_SOA_COLUMN(MeanOccRobust_NtrackDet_Unfm_80, meanOccRobust_NtrackDet_Unfm_80, float);
+
+DECLARE_SOA_COLUMN(WeightMeanOcc_Prim_Unfm_80, weightMeanOcc_Prim_Unfm_80, float);
+DECLARE_SOA_COLUMN(WeightMeanOcc_FV0A_Unfm_80, weightMeanOcc_FV0A_Unfm_80, float);
+DECLARE_SOA_COLUMN(WeightMeanOcc_FV0C_Unfm_80, weightMeanOcc_FV0C_Unfm_80, float);
+DECLARE_SOA_COLUMN(WeightMeanOcc_FT0A_Unfm_80, weightMeanOcc_FT0A_Unfm_80, float);
+DECLARE_SOA_COLUMN(WeightMeanOcc_FT0C_Unfm_80, weightMeanOcc_FT0C_Unfm_80, float);
+DECLARE_SOA_COLUMN(WeightMeanOcc_FDDA_Unfm_80, weightMeanOcc_FDDA_Unfm_80, float);
+DECLARE_SOA_COLUMN(WeightMeanOcc_FDDC_Unfm_80, weightMeanOcc_FDDC_Unfm_80, float);
+
+DECLARE_SOA_COLUMN(WeightMeanOcc_NTrack_PVC_Unfm_80, weightMeanOcc_NTrack_PVC_Unfm_80, float);
+DECLARE_SOA_COLUMN(WeightMeanOcc_NTrack_ITS_Unfm_80, weightMeanOcc_NTrack_ITS_Unfm_80, float);
+DECLARE_SOA_COLUMN(WeightMeanOcc_NTrack_TPC_Unfm_80, weightMeanOcc_NTrack_TPC_Unfm_80, float);
+DECLARE_SOA_COLUMN(WeightMeanOcc_NTrack_TRD_Unfm_80, weightMeanOcc_NTrack_TRD_Unfm_80, float);
+DECLARE_SOA_COLUMN(WeightMeanOcc_NTrack_TOF_Unfm_80, weightMeanOcc_NTrack_TOF_Unfm_80, float);
+DECLARE_SOA_COLUMN(WeightMeanOcc_NTrackSize_Unfm_80, weightMeanOcc_NTrackSize_Unfm_80, float);
+DECLARE_SOA_COLUMN(WeightMeanOcc_NTrackTPC_A_Unfm_80, weightMeanOcc_NTrackTPC_A_Unfm_80, float);
+DECLARE_SOA_COLUMN(WeightMeanOcc_NTrackTPC_C_Unfm_80, weightMeanOcc_NTrackTPC_C_Unfm_80, float);
+DECLARE_SOA_COLUMN(WeightMeanOcc_NTrackITS_TPC_Unfm_80, weightMeanOcc_NTrackITS_TPC_Unfm_80, float);
+DECLARE_SOA_COLUMN(WeightMeanOcc_NTrackITS_TPC_A_Unfm_80, weightMeanOcc_NTrackITS_TPC_A_Unfm_80, float);
+DECLARE_SOA_COLUMN(WeightMeanOcc_NTrackITS_TPC_C_Unfm_80, weightMeanOcc_NTrackITS_TPC_C_Unfm_80, float);
+
+DECLARE_SOA_COLUMN(WeightMeanOccRobust_T0V0Prim_Unfm_80, weightMeanOccRobust_T0V0Prim_Unfm_80, float);
+DECLARE_SOA_COLUMN(WeightMeanOccRobust_FDDT0V0Prim_Unfm_80, weightMeanOccRobust_FDDT0V0Prim_Unfm_80, float);
+DECLARE_SOA_COLUMN(WeightMeanOccRobust_NtrackDet_Unfm_80, weightMeanOccRobust_NtrackDet_Unfm_80, float);
+
+} // namespace trackmeanocc
+
+DECLARE_SOA_TABLE(TrackMeanOccs, "AOD", "TRACKMEANOCCS", o2::soa::Index<>,
+                  o2::aod::trackmeanocc::TrackId,
```

Comment:
The convention for indices in the data model is hierarchical, if something refers to tracks, it should not also refer to the objects that are referred by the tracks themselves - collisions, BCs, etc.